src/fix/main.c

```diff
@@ -967,6 +967,10 @@ static void processFile(int input, int output, char const *name, off_t fileSize)
 
 	if (oldLicensee != UNSPECIFIED)
 		overwriteByte(rom0, 0x14B, oldLicensee, "old licensee code");
+	else if (sgb && rom0[0x14B] != 0x33)
+		fprintf(stderr,
+			"warning: SGB compatibility enabled, but old licensee was %#x, not 0x33\n",
```

Comment:
```suggestion
			"warning: SGB compatibility enabled, but old licensee was 0x%02x, not 0x33\n",
```

Reply:
We have 11 instances of `%#` and 6 instances of `0x%`. Would you like to standardize on one or the other?

Reply:
`%#` is amenable to neither a custom prefix (`$%x`) nor to a clear min width (`0x%02x`), so it shouldn't be used, IMO.